force-app/main/default/classes/UTIL_OrgTelemetry_TEST.cls

```diff
@@ -124,6 +124,21 @@ private class UTIL_OrgTelemetry_TEST {
             UTIL_OrgTelemetry.TelemetryParameterName.IsEnabled_StoreErrors.name(),
             edaSettings.Store_Errors_On__c
         );
+
+        assertBooleanValue(
+            UTIL_OrgTelemetry.TelemetryParameterName.Has_AffiliationsRelatedToDeletedProgramEnrollments.name(),
+            edaSettings.Affl_ProgEnroll_Del__c
+        );
+
+        assertBooleanValue(
+            UTIL_OrgTelemetry.TelemetryParameterName.Has_Affiliation_Start_Date.name(),
+            edaSettings.Affl_ProgEnroll_Copy_Start_Date__c
+        );
+
+        assertBooleanValue(
+            UTIL_OrgTelemetry.TelemetryParameterName.Has_Affiliation_End_Date.name(),
+            edaSettings.Affl_ProgEnroll_Copy_End_Date__c
+        );
```

Comment:
The [T-8132282: EDA Telemetry --> Affiliations](https://gus.lightning.force.com/lightning/r/ADM_Test_Scenario__c/a80EE000000kynZYAQ/view) mentions `validateObjectUsageTelemetry` test method. It is not updated in this PR to contain tests for the Affiliation feature parameters. I would advise to create a new test method to test them instead of adding more code to already long `validateObjectUsageTelemetry` test method. Thoughts?
CC @MatthewBlanski

Reply:
https://github.com/SalesforceFoundation/EDA/pull/1923#discussion_r777750455